doc/custom_rosdistro.rst

```diff
@@ -55,11 +55,19 @@ Add a tag to custom distribution files
 For more flexible configuration options you should always add a ``tag`` to your
 custom distribution file::
 
+    release_platforms:
+      ...
+    repositories:
+      ...
     tags:
     - YOUR_TAG
+    type: distribution
+    version: ...
 
-This allows the build files in the ROS build farm configuration to operate only
-on the packages / repositories from your custom distribution file.
+This allows the build files in the ROS build farm configuration to specify a
```

Comment:
'build files' sounds generic and is difficult to search for context.
I wonder if a glossary would help.
```rst
`build files <glossary.rst#build-files>`_
```

```rst
Glossary
========

build files
-----------
``https://github.com/ros-infrastructure/ros_buildfarm_config/tree/production/ROS_DISTRO/(doc|release|source)-*.yaml``

distribution files
------------------
``https://raw.githubusercontent.com/YOUR_FORK/rosdistro/master/ROS_DISTRO/*.yaml``
```

Reply:
The `doc/configuration_options.rst` page already uses this term and references the repo. And I would expect that it has been read before setting up a custom distro. Anyway I am probably not a good person to ask how to improve the docs.

Please feel free to open pull requests to improve the docs in any way.